build-tools/automation/azure-pipelines-nightly.yaml

```diff
@@ -185,9 +185,9 @@ stages:
     strategy:
       parallel: 3
     pool:
-      name: VSEng-VSMac-Xamarin-Shared
+      name: $(SharedMacPool)
       demands:
-      - macOS.Name -equals Ventura
+      - macOS.Name -equals $(SharedMacName)
       - macOS.Architecture -equals x64
```

Comment:
I assume we are intentionally keeping the tests on `x64`?

Out of curiosity, did you try `arm64`? Was there an issue? Were they faster/slower?

Reply:
I'm planning to revisit this for the MSBuild and emulator test jobs on both our regular CI and nightly. I don't have the full set of issues I've ran into with different machines in the past but wanted to leave this unchanged for now and look at upgrading most of the test stages separately.